### PR TITLE
testnet_v1: add bridged-token-list.json with UAN

### LINKED ISSUE #44: update bridged-token-list for Godowken testnet_v1
image
Some deployed sUDT_ERC20_Proxy [contracts](https://github.com/nervosnetwork/light-godwoken/blob/490c11e65f/src/light-godwoken/constants/tokens.ts#L56-L102) on [Godwoken testnet token bridge](https://testnet.bridge.godwoken.io/#/v1)

- [x] Add `testnet_v1/bridged-token-list.json` containing [UAN](https://github.com/nervosnetwork/rfcs/pull/335) and displayName
- [x] GwScan will register the above tokens to https://v1.testnet.gwscan.com/tokens/bridge

## References
- https://github.com/nervosnetwork/light-godwoken/blob/490c11e65f/src/light-godwoken/constants/tokens.ts#L56-L102
- https://github.com/nervosnetwork/godwoken-info/pull/41
- [mainnet_v1/bridged-token-list.json](https://github.com/nervosnetwork/godwoken-info/blob/rfc-uan/mainnet_v1/bridged-token-list.json)

### PR DESCRIPTION
**Pull request by ShookLyngs**
Previous PR was closed mistakenly: 
- https://github.com/nervosnetwork/godwoken-info/pull/47

These are the bridged tokens on Godwoken testnet_v1.
I have referenced with these files, please correct me if my commited info is wrong:

- https://github.com/nervosnetwork/light-godwoken/blob/490c11e65f/src/light-godwoken/constants/tokens.ts#L56-L102
- [mainnet_v1/bridged-token-list.json](https://github.com/nervosnetwork/godwoken-info/blob/main/mainnet_v1/bridged-token-list.json)
- https://github.com/pygman/force-bridge/blob/uan-token-list/configs/testnet-all-bridged-tokens.json

### CODE REVIEW

File: testnet_v1_1/bridged-token-list.json [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      "symbol": "ETH"
+    },
+    "erc20Info": {
+      "sudtScriptArgs": "0x1b072aa0ded384067106ea0c43c85bd71bafa5afdb432123511da46b390a4e33",
+      "ethAddress": "0xf0d66bf1260D21fE90329A7A311e84979FEB004d"
+    },
+    "UAN": "ETH.gw|gb.ckb|fb.eth",
+    "displayName": "ETH (via ForceBridge from ETH)"
+  }
+]
```
Comment (Flouse) [inline testnet_v1_1/bridged-token-list.json]:
```suggestion
]

```

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
-chain-data/
\ No newline at end of file
+chain-data/
+.idea/
```
Comment (Flouse) [inline .gitignore]:
```suggestion
.idea/

```

**Review by Flouse — COMMENTED** [review]
suggestion: add new line to the end of file.

**Comment by Flouse** [comment]
> I have referenced with these files, please correct me if my commited info is wrong:

## Check-list

- [x] [pCKB.gw|gb.ckb](https://gw-explorer.nervosdao.community/address/0xE05d380839f32bC12Fb690aa6FE26B00Bd982613/read-contract)
- [x] [GWK.gw|gb.ckb](https://gw-explorer.nervosdao.community/address/0x2275afe815de66beabe7a2c03005537ab843afb2/read-contract)
- [x] [TTKN.gw|gb.ckb](https://gw-explorer.nervosdao.community/address/0x3c41EDd22658b2D6Cf0Eb26da941d74fe45BeA52/read-contract) 
- [x] [DAI.gw|gb.ckb|fb.eth](https://gw-explorer.nervosdao.community/address/0x24a9467Fd390d8ca70e848Ce8a2E9BbE087Eab0E/read-contract)
- [x] [USDC.gw|gb.ckb|fb.eth](https://gw-explorer.nervosdao.community/address/0x20FB98bb94aD9B98c0f0089138012E49323d0fEf/read-contract)
- [x] [USDT.gw|gb.ckb|fb.eth](https://gw-explorer.nervosdao.community/address/0x90fc553aBad2b8B6ffe9282e36db52cE6388C648/read-contract)
- [x] [ETH.gw|gb.ckb|fb.eth](https://gw-explorer.nervosdao.community/address/0xf0d66bf1260D21fE90329A7A311e84979FEB004d/read-contract)

> ## Reference
> - https://github.com/nervosnetwork/rfcs/pull/335

**Review by Flouse — APPROVED** [review]

File: testnet_v1_1/bridged-token-list.json (hunk 1 above)
Comment (ShookLyngs) [reply testnet_v1_1/bridged-token-list.json]: Okay, but what is the reason of doing this?

Comment (Flouse) [reply testnet_v1_1/bridged-token-list.json]: image

> If content is added to the end of the file, then the line that was previously the last line will have been edited to include a newline character. This means that blame ing the file to find out when that line was last edited will show the text addition, not the commit before that you actually wanted to see.

see: https://stackoverflow.com/a/729795/7365437

You can configure your editor to do this automatically.

**Review by homura — APPROVED** [review]
LGTM

**Comment by Flouse** [comment]
image
https://v1.testnet.gwscan.com/tokens/bridge updated.

**Pull request merged**